anmelde_tool/registration/views.py

```diff
@@ -234,6 +234,38 @@ def participant_initialization(self, request) -> Registration:
 
         return registration
 
+class CheckPersonViewSet(viewsets.ReadOnlyModelViewSet):
+
+    def participant_initialization(self, request) -> Registration:
+        input_serializer = (
+            registration_serializers.RegistrationParticipantPutSerializer(
+                data=request.data
+            )
+        )
+        input_serializer.is_valid(raise_exception=True)
+
+        registration_id = self.kwargs.get("registration_pk", None)
+        registration: Registration = get_registration(registration_id)
+
+        return registration
+
+    @staticmethod
+    def check_for_double_participants(request, event_id):
+        if RegistrationParticipant.objects.filter(
+                first_name=request.data.get("first_name"),
+                last_name=request.data.get("last_name"),
+                birthday=request.data.get("birthday"),
+                registration__event=event_id,
+        ).exists():
+            raise ParticipantAlreadyExists()
+
+    def create(self, request, *args, **kwargs) -> Response:
+        event_id = self.kwargs.get("registration_pk", None)
+
+        self.check_for_double_participants(request, event_id)
+
+        return Response('Person ist nicht vorhanden.', status=status.HTTP_200_OK)
```

Comment:
Verstehe ich nicht. Warum ein create method in einem ReadOnlyModelView?

Reply:
wie denn sonst?
haben wir doch schon vorher so gemacht?

<img width="700" alt="image" src="https://github.com/dpvonline/scout-tool-backend/assets/54767884/d6dc2ada-51a0-412a-8bc2-235c2d8f3d0a">